cli/integration/tests/waiter/util.py

```diff
@@ -143,3 +144,20 @@ def wait_until_inner():
             details = str(final_response)
         logging.info(f"Timeout exceeded waiting for condition. Details: {details}")
         raise
+
+
+def load_json_file(path):
+    """Decode a JSON formatted file."""
+    content = None
+
+    if os.path.isfile(path):
+        with open(path) as json_file:
+            try:
+                logging.debug(f'attempting to load json configuration from {path}')
+                content = json.load(json_file)
+            except Exception:
+                pass
```

Comment:
- We should log that there was an error reading the content as json.
- Is returning `None` handled well upstream by the caller?

Reply:
I added a log when the JSON file could not be loaded, and I made the caller handle `None` more gracefully